client/src/components/thread/components/add-comment/add-comment.jsx

```diff
@@ -14,17 +16,23 @@ const AddComment = ({ postId, onCommentAdd }) => {
     setBody('');
   };
 
+  const handleTextChange = useCallback(
+    ev => setBody(ev.target.value),
+    [setBody]
+  );
+
   return (
-    <Form reply onSubmit={handleAddComment}>
-      <Form.TextArea
+    <>
+      <TextArea
+        className={styles.commentArea}
         value={body}
         placeholder="Type a comment..."
-        onChange={ev => setBody(ev.target.value)}
+        onChange={handleTextChange}
       />
-      <Button type={ButtonType.SUBMIT} isPrimary>
+      <Button type={ButtonType.BUTTON} onClick={handleAddComment} isPrimary>
```

Comment:
```suggestion
      <Button type={ButtonType.SUBMIT} isPrimary>
```
set listener on form on `onSubmit` event